README.md

```diff
@@ -8,6 +8,7 @@ const method = new ValidatedMethod({
   name, // DDP method name
   mixins, // Method extensions
   validate, // argument validation
+  callOptions, // options passed to Meteor.apply
```

Comment:
Hmm.. I'd say rename to `applyOptions`, since it's inherently linked to that part of the Meteor API. Speaking of which, the docs for Meteor.apply don't mention any of these options: http://docs.meteor.com/#/full/meteor_apply

We should fix that.